app/views/deploys/new.html.erb

```diff
@@ -40,6 +40,8 @@
         </div>
       </div>
 
+      <%= Samson::Hooks.render_views(:deploy_new, self, project: @project, form: form, stage: @stage) %>
+
       <%= Samson::Hooks.render_views(:deploy_form, self, project: @project, form: form) %>
```

Comment:
why not put this into the deploy_form ?
... can add the `stage` to that hook, it's just an options hash